src/tests/system/tests/test_ldap.py

```diff
@@ -14,173 +14,102 @@
 from sssd_test_framework.topology import KnownTopology
 
 
-@pytest.mark.ticket(bz=[795044, 1695574])
-@pytest.mark.importance("critical")
-@pytest.mark.parametrize("modify_mode", ["exop", "ldap_modify", "exop_force"])
-@pytest.mark.parametrize("use_ppolicy", ["true", "false"])
-@pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
 @pytest.mark.topology(KnownTopology.LDAP)
-@pytest.mark.require(
-    lambda client, sssd_service_user: ((sssd_service_user == "root") or client.features["non-privileged"]),
-    "SSSD was built without support for running under non-root",
-)
```

Comment:
Do we not need/want to run these test cases with both root and the sssd service user?

Reply:
I don't think it's important, it's passing the request to the directory server, AFAIK, there is no need for privilege escalation. If dev comes back and says it's important, we'll add it back in.